compute/disk-images/disk_image_test.go

```diff
@@ -121,8 +193,43 @@ func TestComputeDiskImageSnippets(t *testing.T) {
 		}
 	})
 
+	t.Run("Test image creation and deletion from snapshot", func(t *testing.T) {
+		disk, err := getDisk(ctx, tc.ProjectID, diskName, zone)
+		if err != nil {
+			t.Fatalf("getDisk got err: %v", err)
+		}
+
+		err = createSnapshot(ctx, tc.ProjectID, snapshotName, disk, &[]string{})
+		if err != nil {
+			t.Fatalf("getDisk got err: %v", err)
+		}
+
+		want := "created"
+		if err := createImageFromSnapshot(buf, tc.ProjectID, snapshotName, imageName, []string{}, guestOsFeatures); err != nil {
+			t.Fatalf("createImageFromDisk got err: %v", err)
```

Comment:
this needs to be a `Errorf`, to allow the delete to run. otherwise test failures here will orphan the snapshot resource.